frontend/app/src/components/defi/Borrowing.vue

```diff
@@ -7,50 +7,72 @@
   <v-container v-else>
     <v-row>
       <v-col cols="12">
-        <h2>MakerDAO Vaults</h2>
+        <h2>Collateralized Loans</h2>
       </v-col>
     </v-row>
     <v-row>
-      <v-col>
-        <stat-card title="Total collateral locked">
-          <amount-display
-            :value="makerDAOVaultSummary.totalCollateralUsd"
-            show-currency="symbol"
-            fiat-currency="USD"
-          ></amount-display>
-        </stat-card>
-      </v-col>
-      <v-col>
-        <stat-card title="Total debt">
-          <amount-display
-            :value="makerDAOVaultSummary.totalDebt"
-            asset="DAI"
-          ></amount-display>
-        </stat-card>
+      <v-col cols="12">
+        <stat-card-wide cols="two">
+          <template #first-col>
+            <dl>
+              <dt class="title font-weight-regular">
+                Total collateral locked
+              </dt>
+              <dd class="primary--text headline font-weight-bold">
+                <amount-display
+                  :value="makerDAOVaultSummary.totalCollateralUsd"
+                  show-currency="symbol"
+                  fiat-currency="USD"
+                ></amount-display>
+              </dd>
+            </dl>
+          </template>
```

Comment:
are all this contents follow the same strucuture inside?

Reply:
I was thinking maybe we could extract a sub-component here to avoid duplication

Reply:
A stat card will not necessarily always have content following this exact structure, but it might. I guess we could extract it as a basic usage?

Reply:
I am closing at the moment but if we have some standard structure for the columns maybe we could evaluate making a component out of it

Reply:
Perhaps we can have an issue for this? Just so it's not forgotten? Can you make one when you find some time @isidorosp ?